src/DonationForms/Endpoints/ListDonationForms.php

```diff
@@ -176,7 +177,11 @@ public function getForms(): array
      */
     public function getTotalFormsCount(): int
     {
-        $query = give()->donationForms->prepareQuery();
+        $builder = new ModelQueryBuilder(DonationForm::class);
```

Comment:
If you're not using the models for these, then you should not be using the `ModelQueryBuilder`.  Instead you should just use the regular `QueryBuilder` like this:

`DB::table('posts')->where()`

Reply:
Resolved: e13dd57ce3cc1184463bf52711077f0c3b212542